src/Frontend/Modules/Location/Layout/Templates/Index.html.twig

```diff
@@ -25,9 +25,7 @@
 
             <div itemprop="name" class="hidden"><strong>{{ locationItem.title }}</strong></div>
             <address itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
-              <span itemprop="streetAddress">{{ locationItem.street }} {{ locationItem.number }}</span><br>
-              <span itemprop="postalCode">{{ locationItem.zip }}</span>
-              <span itemprop="addressLocality">{{ locationItem.city }}</span><br>
+              <span itemprop="streetAddress">{{ locationItem.address }}</span><br>
```

Comment:
That is because it isn't advised to put the entire address in one address field.
Zip, city and country should always be separate.
![schermafbeelding 2017-06-13 om 08 32 40](https://user-images.githubusercontent.com/588616/27068971-eaade3e2-5012-11e7-99d6-9af8ef74e8df.png)